src/QuizContent.js

```diff
@@ -0,0 +1,204 @@
+import React, { useState, useEffect } from 'react';
+import {Container, Typography, RadioGroup, FormControlLabel, Radio, Button} from '@mui/material';
+import { Box } from '@mui/system';
+import Grid from '@mui/material/Unstable_Grid2'
+
+
+function QuestionContent({ question, options, onAnswerSelected, onSkipQuestion, onBackQuestion}) {
+    const [selected, setSelected] = useState('');
+    const resetSelection = () => {
+        setSelected('');
+    }
+
+    return (
+        <Container>
+            <Typography variant="h5" align="left" sx = {{pl:2, pb:2}}>{question}</Typography>
+            <RadioGroup
+                value={selected}
+                onChange={(e) => setSelected(e.target.value)}
+                style = {{
+                    paddingLeft:"10px"
+                }}
+                id = "answer-choices"
```

Comment:
Please, remove spaces before and after the `=` sign.

Reply:
Prettier will do it for me!